### PR TITLE
disable browse button if no access role

### PR DESCRIPTION
**Pull request by djamnazi**
[OO-BUGFIX] Remove browse button if user doesn't have the access to the media management.

### CODE REVIEW

File: MediaAdminBundle/Resources/views/Form/form_div_layout.html.twig [hunk 1]
```diff
@@ -20,21 +20,27 @@
             />
             <span class="media-form-no-img-selected">{{ 'open_orchestra_media_admin.form.media.no_media_selected'|trans }}</span>
         </div>
-        {% if not disabled %}
-            <div class="media-form-type">
-                <a class="btn btn-default mediaModalOpen"
+        <div class="media-form-type">
+            {% if not disabled   %}
+                <a class="btn btn-default mediaModalOpen
+                    {% if not is_granted('ROLE_ACCESS_MEDIA_FOLDER') %} disabled hidden {% endif %}"
```
Comment (alavieille) [inline MediaAdminBundle/Resources/views/Form/form_div_layout.html.twig]: you must pass the document folder and test than work with tree folder roles

File: MediaAdminBundle/Resources/views/Form/form_div_layout.html.twig [hunk 2]
```diff
@@ -20,21 +20,27 @@
...
                     <i class="fa fa-folder-open-o"></i>
                     {{ 'open_orchestra_media_admin.left_menu.editorial.browse'|trans({'language':context.currentLocale}) }}
                 </a>
                 <br /><br />
-                <button class="btn btn-warning clear-media" data-input="{{ id }}">
-                    {{ 'open_orchestra_backoffice.form.remove'|trans }}
-                </button>
-            </div>
-        {% endif %}
+                 {% if not is_granted('ROLE_ACCESS_MEDIA_FOLDER') %}
```
Comment (alavieille) [inline MediaAdminBundle/Resources/views/Form/form_div_layout.html.twig]: same

**Comment by alavieille** [comment]
i think you are a problem with a rebase

**Comment by djamnazi** [comment]
i'm fixing this

**Pull request closed without merging**